maven-plugins/hk2-inhabitant-generator/src/main/java/org/jvnet/hk2/generator/maven/AbstractInhabitantsGeneratorMojo.java

```diff
@@ -57,9 +57,9 @@ public abstract class AbstractInhabitantsGeneratorMojo extends AbstractMojo {
     private boolean verbose;
 
     /**
-     * @parameter default-value=true
+     * @parameter default-value="true"
      */
-    private final boolean includeDate = true;
+    private boolean includeDate;
```

Comment:
Javadoc - isn't default false now?

Reply:
no, because the injection of default value is done by Maven plugin management = what was blocked by the final modifier
see previous parameters that are Strings, where the default value is injected: it works as well for booleans

Reply:
Aha, the old version didn't update it at all, and new version now uses default from the `@parameter` annotation.